instrumentation/logback/logback-mdc-1.0/testing/src/main/java/io/opentelemetry/instrumentation/logback/mdc/v1_0/LogbackInstrumentationTest.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.logback.mdc.v1_0;
+
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+
+public interface LogbackInstrumentationTest {
+  static InstrumentationExtension agentTesting = AgentInstrumentationExtension.create();
```

Comment:
I think it's clearer if you put it in the javaagent tests that actually use that; even if it results in a bit more code & duplication.